tests/integration/iop-remote-integration-test-defaults.yaml

```diff
@@ -0,0 +1,30 @@
+# This file provides some defaults for integration testing.
```

Comment:
should we use remote profile here in the test, similarly as what we tell users to do?

Reply:
remote profile is empty for now. 
```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec: {}
```
Should we update it to use the version like here?

Reply:
Yes i was surprised to find this remote profile is wrong.  I think it is a just placeholder now. should be a profile that has pilot disabled.

Reply:
ok to address this comment as a follow up PR. :)